src/userent.c

```diff
@@ -237,38 +238,117 @@ struct user_entry_type USERENTRY_INFO = {
   def_tcl_append
 };
 
+int pass2_set(struct userrec *u, struct user_entry *e, void *new)
+{
+  if (e->u.extra) {
+    explicit_bzero(e->u.extra, strlen(e->u.extra));
+    nfree(e->u.extra);
+  }
+  if (new) { /* set PASS2 */
+    e->u.extra = user_malloc(strlen(new) + 1);
+    strcpy(e->u.extra, new);
+  }
+  else /* remove PASS2 */
+    e->u.extra = new;
+  return 0;
+}
+
+static int def_tcl_null(Tcl_Interp * irp, struct userrec *u,
+                        struct user_entry *e, int argc, char **argv)
+{
+  Tcl_AppendResult(irp, "Please use PASS instead.", NULL);
+  return TCL_OK;
```

Comment:
return TCL_ERROR; should be better, as it raises an error on the Tcl level, TCL_OK does not and might never show this error

Reply:
Fixed with commit fd3d0a5d715439954827555c611efdaeb3ca08cc